packages/terra-date-picker/package.json

```diff
@@ -31,6 +31,7 @@
     "focus-trap-react": "^6.0.0",
     "keycode-js": "^2.0.1",
     "moment": "^2.21.0",
+    "moment-timezone": "^0.5.13",
```

Comment:
moment-timezone is an add-on for moment so we should be able to remove the moment dependency here and [here](https://github.com/cerner/terra-framework/blob/88d3580c37b3965d3b563623e6a8d4843a913d85/packages/terra-date-time-picker/package.json#L32) in terra-date-time-picker. Anywhere we have `import moment from 'moment'` should be replaced with `import moment from 'moment-timezone'`.

Here are some references on why we don't need both:
- https://stackoverflow.com/questions/49729646/using-both-moment-and-moment-tz
- https://thecodebarbarian.com/a-practical-guide-to-moment-timezone.html

Reply:
Changed here 11b765b40bac98b52e15037a385965c7f9713921